openapi/data_repository_service.swagger.yaml

```diff
@@ -353,6 +353,25 @@ definitions:
           Name of the region in the cloud service provider that the object belongs to.
         example:
           us-east-1
+      provider:
+        type: string
+        description: >-
+          The cloud service provider. This provided the explicit scope of the
+          `region` field.
+        enum:
+          - s3
+          - gs
+      region_locked:
```

Comment:
There are some discussions going on that might make this one something to hold in reserve. On the other hand, it can be ignored later on.